apps/dashboard/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "preinstall": "npx only-allow pnpm",
-    "dev:no-cache": "rm -rf .next/cache && pnpm run dev",
+    "dev:no-cache": "git clean -xdf && pnpm run dev",
```

Comment:
doesnt that remove all the untracked files? like even my .envs and stuff?

Reply:
This was removed in the upstream change, but it only removes untracked files that are not in .gitignore so shouldnt touch .env